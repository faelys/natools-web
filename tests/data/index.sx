(elements
   (title Index)
   (main-text
      (tags
         (pages
            "<ul>\n"
            (all-elements
               (render "<li>" (render href) "</li>\n"))
            "</ul>\n<p>Tags:"
            (tags tags (all-children (template
               "\n<a href=\"/tags/" (name (2)) "\">" (name (2)) "</a>")))
            "</p>\n"))
      (load-date "<p>Data loaded on " (in-zone 0 (rfc-3339)) "</p>\n"))
   (sidebar ""))
