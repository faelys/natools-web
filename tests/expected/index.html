<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
  <head>
    <title>Index</title>
  </head>
  <body>
<h1>Index</h1>
<ul>
<li><a href="/first">First Page</a></li>
<li><a href="/second">Second Page</a></li>
<li><a href="/third">Third Page</a></li>
<li><a href="/fourth">Fourth Page</a></li>
<li><a href="/fifth">Fifth Page</a></li>
</ul>
    <div class="sidebar">
    </div>
    <div class="footer">
<p>Default footer: <a href="/index">return to index</a></p>
    </div>
  </body>
</html>
